web/src/components/tasks/business-formation/getErrorStateForFormationField.ts

```diff
@@ -283,8 +283,8 @@ export const getErrorStateForFormationField = (inputParams: {
     }
 
     const partialAddressError = fieldWithAssociatedFields({
-      associatedFields: ["addressMunicipality", "addressLine1"],
-      label: Config.formation.general.partialAddressErrorText,
+      associatedFields: ["addressMunicipality", "addressLine1", "addressLine2"],
```

Comment:
[sand] It looks like all of these also depend on `addressLine2` for validation. Should there be a test that checks if `addressLine1`, `addressMunicipality`, and `addressZipCode` all have an error if `addressLine2` has a value?

If this isn't how the app behaves then you can ignore this.

Reply:
done added a test